src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/process/DefaultProcessRunner.java

```diff
@@ -77,19 +78,15 @@ public void run(String[] command) throws ProcessRunnerException {
       }
       processBuilder.command(makeOsSpecific(command));
 
-      synchronized (this) {
```

Comment:
Yeah maybe we should separate out the cloudsdk configuration and the cloud sdk execution. It seems a little dangerous to re-use an execution environment, no?